Database/database_scripts.sql

```diff
@@ -0,0 +1,65 @@
+/* This file can be run to create the necessary objects for
+    a Dao game database */
+
+/* Create the Database */
+CREATE DATABASE DAOdb;
+GO
+
+/* Create database objects schemas */
+USE DAOdb;
+GO
+CREATE SCHEMA game
+CREATE SCHEMA plyr
+CREATE SCHEMA fun

```

Comment:
Are creating these schemas like declaring a variable?  Or is it like creating a name that you can reference and ties stuff together?

Reply:
Schemas help you organize a database. For our DAO db, they aren't really necessary, but for larger databases it allows you to organize and probably most importantly can be set up with permissions. 
I set up DAO db with these three to start, 

It'll look like this:
DAOdb:
       fun.fun_positions
       ...
       game.history_log
       game.game_results
       ...
       plyr.players
       ...
       
So when you reference a table or a function or whatever, the full path will include the schema. If I wanted to, I could assign different login credentials to different user rights groups and allow access to each schema separately.